localstack/services/lambda_/invocation/lambda_service.py

```diff
@@ -549,12 +550,26 @@ def store_lambda_archive(
     )
 
 
-def create_hot_reloading_code(path: str) -> HotReloadingCode:
-    # TODO extract into other function
-    if not PurePosixPath(path).is_absolute() and not PureWindowsPath(path).is_absolute():
+def assert_hot_reloading_path_absolute(path: str) -> None:
+    """
+    Check whether a given path is an absolute path, either posix or windows.
```

Comment:
nit: it checks whether it can be expanded into an absolute path, but does not need to be an absolute path itself 💡

Reply:
Maybe, `assert_hot_reloading_path_absolute_resolves_absolute` ?

Reply:
fair point, let's leave the method. I just got confused by "Check whether a given path is an absolute path"
because `$HEAD_DIR/my/relative/suffix/path` is an invalid path by itself but is valid if it resolved to an absolute path

Reply:
Hmm, I get it now. Will adjust the docstring to make that clearer.